api/models.py

```diff
@@ -62,3 +62,17 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event(models.Model):
+
+	title = models.CharField(max_length=255, blank=False, null=False)
+	start_date = models.DateField(blank=False, null=False)
```

Comment:
but I do think that part is fixed now, though...
I just don't like the "that's what I did but you/someone/whoever said", when that is not true :wink: